studiopress/genesis-custom-blocks
# Add Genesis Pro upgrade page

#### Changes
* Re-enable the Pro nag
* Change the Pro screen from the previous Block Lab Pro upsell to Genesis Pro
* Rename the menu item

#### Testing instructions
* Check that the Genesis Pro menu exists
* Check that the page resembles this: https://gist.github.com/RobStino/31886f0156f244c3e0dc86df820af3d0
* Check that all the images load
